src/fetchImage.tsx

```
@@ -0,0 +1,31 @@
+import { doc, getDoc } from "firebase/firestore";
+import { ref, getDownloadURL } from "firebase/storage";
+import { getStorage } from "firebase/storage";
+import { getFirestore } from "firebase/firestore";
+
+const storage = getStorage();
+const db = getFirestore();
+
+const fetchImage = async (userUid: string) => {
```

ESLint

[error] 'fetchImage' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)